src/script/auth/module/reducer/authReducer.ts

```diff
@@ -43,6 +42,11 @@ export interface RegistrationDataState {
   termsAccepted: boolean;
 }
 
+export interface LoginDataState {
+  clientType: ClientType;
+  phone?: string;
+}
```

Comment:
```suggestion
export type LoginDataState = Pick<LoginData, 'clientType' | 'phone'>.
```

Would this work? Or are they slightly different?

Reply:
Works fine, I like this approach 🚀